.github/actions/test/action.yml

```diff
@@ -24,8 +24,18 @@ runs:
 
     - name: clean up
       run: |
-          echo "cleaning up wheel files..."
+          echo "cleaning up disk space..."
           find . -type f -name '*.whl' -exec rm -rf {} \;
+          rm -rf ~/.cache/pip
+          sudo rm -rf /usr/local/.ghcup
+          sudo rm -rf /opt/hostedtoolcache/CodeQL
```

Comment:
I don't know the complete purpose of clearing this, but worry that this command is using too much brute force.  Would it be better to use the CodeQL tools to clean up? https://docs.github.com/en/code-security/codeql-cli/codeql-cli-manual/database-cleanup